app/tests/test_item_model.py

```diff
@@ -127,3 +128,50 @@ def test_item_property_eo_gsd(self):
         self.assertEqual(
             item.properties_eo_gsd, min(eo_gsd), msg='eo:gsd is not the min value from asset'
         )
+
+    def test_item_create_model_valid_geometry(self):
+        # a correct geometry should not pose any problems
+        item = Item(
+            collection=self.collection,
+            properties_datetime=utc_aware(datetime.utcnow()),
+            item_name='item-1', geometry=GEOSGeometry(
+                'SRID=4326;POLYGON '
+                '((5.96 45.82, 5.96 47.81, 10.49 47.81, 10.49 45.82, 5.96 45.82))'
+            )
+        )
+        item.full_clean()
+
+    def test_item_create_model_invalid_geometry(self):
+        # a geometry with self-intersection should not be allowed
+        with self.assertRaises(ValidationError):
+            item = Item(
+                collection=self.collection,
+                properties_datetime=utc_aware(datetime.utcnow()),
+                item_name='item-1', geometry=GEOSGeometry(
+                    'SRID=4326;POLYGON '
+                    '((5.96 45.82, 5.96 47.81, 10.49 45.82, 10.49 47.81, 5.96 45.82))'
+                )
+            )
+            item.full_clean()
+
+    def test_item_create_model_empty_geometry(self):
+        # empty geometry should not be allowed
+        with self.assertRaises(ValidationError):
+            item = Item(
+                collection=self.collection,
+                properties_datetime=utc_aware(datetime.utcnow()),
+                item_name='item-empty', geometry=GEOSGeometry(
+                        'POLYGON EMPTY'
+                )
```

Comment:
code style, don't forget to run `make format` or even better integrate yapf in your IDE